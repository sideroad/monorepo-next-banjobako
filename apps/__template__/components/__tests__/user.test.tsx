import React from 'react';
import User from '../user';
import renderer from 'react-test-renderer';

test('User to render submit form', () => {
  const component = renderer.create(
    <User
      login="sideroad"
      avatar_url="https://avatars2.githubusercontent.com/u/411486?v=4"
      html_url="https://github.com/sideroad"
    />
  );
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
