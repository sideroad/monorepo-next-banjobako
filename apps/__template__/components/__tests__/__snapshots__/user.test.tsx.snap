// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`User to render submit form 1`] = `
<div
  className="jsx-2202639528"
  data-cid="2YoDeJcK"
>
  <div
    className="jsx-2202639528 avator"
    style={
      Object {
        "backgroundImage": "url(https://avatars2.githubusercontent.com/u/411486?v=4)",
      }
    }
  />
  <p
    className="jsx-2202639528"
  >
    <a
      className="jsx-2202639528"
      href="https://github.com/sideroad"
    >
      sideroad
    </a>
  </p>
</div>
`;
